src/pubtools/_marketplacesvm/tasks/delete/command.py

```diff
@@ -227,24 +227,15 @@ def add_args(self):
     def run(self, collect_results: bool = True, allow_empty_targets: bool = False) -> RUN_RESULT:
         """Execute the delete command workflow."""
         mapped_items = [x for x in self.raw_items]
-        executor = Executors.thread_pool(
-            name="pubtools-marketplacesvm-delete",
-            max_workers=min(max(len(mapped_items), 1), self._REQUEST_THREADS),
-        )
```

Comment:
I assume we decided to remove threadpool because:
* operation is not slow a publish/upload :-)
* it is more deterministic

If yes than please resole my comment :-)

Reply:
Yeah, it seemed like added complication to something that doesn't require a lot of time to complete.